src/bindings/python/requirements.txt

```diff
@@ -1,2 +1 @@
-numpy==1.19.4
-typing; python_version < '3.6'
```

Comment:
Quite a lot of files in `src/bindings` import symbols from this module. Is it ok to remove `typing` from here?

Reply:
Starting with Python 3.5 `typing` module is part of stdlib: https://pypi.org/project/typing/, whereas OpenVINO support Python 3.6-3.9.